packages/feature-flags/src/index.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @typedef {Function} Logger
+ * @param {...string} args
+ * @returns {void}
+ * */
+
+/**
+ * @param {{ info: Logger, error: Logger }} logger
+ * @param {string} [endpoint]
+ * @returns {(featureFlagName: string, localPlanningAuthorityCode?: string) => Promise<boolean>}
+ * */
+export const isFeatureActive =
+	(logger, endpoint) => async (featureFlagName, localPlanningAuthorityCode) => {
```

Comment:
We don't need `localPlanningAuthorityCode` this for applications, LPAs aren't involved. We may want a more generic name, and still allow checking user group?

Reply:
Ah right ok, sure I'll rename and leave it as an optional argument.